protocols/CopyToResultsDir.sh

```diff
@@ -53,68 +53,68 @@ mkdir -p "${projectResultsDir}/qcmetrics"
+	rsync -av "${intermediateDir}"/*.quality_by_cycle.pdf "${projectResultsDir}/qcmetrics"
+	rsync -av "${intermediateDir}"/*.quality_distribution.pdf "${projectResultsDir}/qcmetrics"
+	rsync -av "${intermediateDir}"/*.quality_distribution_metrics "${projectResultsDir}/qcmetrics"
+	rsync -av "${intermediateDir}"/*.base_distribution_by_cycle.pdf "${projectResultsDir}/qcmetrics"
+	rsync -av "${intermediateDir}"/*.base_distribution_by_cycle_metrics "${projectResultsDir}/qcmetrics"
+	rsync -av "${intermediateDir}"/*.alignment_summary_metrics "${projectResultsDir}/qcmetrics"
+        rsync -av "${intermediateDir}"/*.flagstat "${projectResultsDir}/qcmetrics"
+	rsync -av "${intermediateDir}"/*.idxstats "${projectResultsDir}/qcmetrics"
+	rsync -av "${intermediateDir}"/*.collectrnaseqmetrics "${projectResultsDir}/qcmetrics"
 
 	if [ "${seqType}" == "PE" ]
         then
-		cp "${intermediateDir}"/*.insert_size_metrics "${projectResultsDir}/qcmetrics"
+		rsync -av "${intermediateDir}"/*.insert_size_metrics "${projectResultsDir}/qcmetrics"
 	else
 		echo "Skip insertSizeMetrics. seqType is: ${seqType}"
 	fi
 
 # copy GeneCounts to results directory
 
-	cp "${intermediateDir}"/*.counts.txt "${projectResultsDir}/expression/"
-	cp "${annotationGtf}" "${projectResultsDir}/expression/"
-	cp "${projectHTseqExpressionTable}" "${projectResultsDir}/expression/"
+	rsync -av "${intermediateDir}"/*.counts.txt "${projectResultsDir}/expression/"
+	rsync -av "${annotationGtf}" "${projectResultsDir}/expression/"
+	rsync -av "${projectHTseqExpressionTable}" "${projectResultsDir}/expression/"
 
 # copy Deseq2 results to results directory
-	cp "${intermediateDir}"/*deseq2_* "${projectResultsDir}/expression/deseq2/"
-	cp "${intermediateDir}"/*design.csv "${projectResultsDir}/expression/deseq2/"
-	cp "${intermediateDir}"/*.svg "${projectResultsDir}/expression/deseq2/"
+	rsync -av "${intermediateDir}"/*deseq2_* "${projectResultsDir}/expression/deseq2/"
+	rsync -av "${intermediateDir}"/*design.csv "${projectResultsDir}/expression/deseq2/"
+	rsync -av "${intermediateDir}"/*.svg "${projectResultsDir}/expression/deseq2/"
 
 # Copy QC images and report to results directory
 
-	cp "${intermediateDir}"/*.collectrnaseqmetrics.pdf "${projectResultsDir}/qcmetrics/"
+	rsync -av "${intermediateDir}"/*.collectrnaseqmetrics.pdf "${projectResultsDir}/qcmetrics/"
 
 # Copy variant vcfs.
 
-        cp "${projectBatchGenotypedVIPPrefix}"* "${projectResultsDir}/variants/vip/"
-	cp "${projectBatchGenotypedVariantCalls}"* "${projectResultsDir}/variants/"
+        rsync -av "${projectBatchGenotypedVIPPrefix}"* "${projectResultsDir}/variants/vip/"
+	rsync -av "${projectBatchGenotypedVariantCalls}"* "${projectResultsDir}/variants/"
 
 # Copy leafcutter
-	cp "${intermediateDir}"*.leafcutter.outlier* "${projectResultsDir}/leafcutter/"
+	rsync -av "${intermediateDir}"*.leafcutter.outlier* "${projectResultsDir}/leafcutter/"
```

Comment:
I would put a slash behind the ${intermediateDir}, just to be sure

Reply:
done